float64[] l_wrist_pos
float64[] r_wrist_pos
float64[] l_wrist_quat
float64[] r_wrist_quat
float64[] l_elbow_pos
float64[] r_elbow_pos
float64[] l_glove_angle
float64[] r_glove_angle